tools/dgeni/templates/constant.template.html

```diff
@@ -15,10 +15,12 @@ <h4 id="{$ constant.name $}" class="docs-header-link docs-api-h4 docs-api-consta
   <p class="docs-api-constant-description">{$ constant.description | marked | safe $}</p>
 {%- endif -%}
 
+<div class="docs-markdown">
 <pre class="docs-markdown-pre">
 <code class="docs-markdown-code">
 {%- highlight "typescript" -%}
   const {$ constant.name | safe $}: {$ constant.type | safe $};
 {%- end_highlight -%}
 </code>
 </pre>
+</div>
```

Comment:
The indentation seems amiss here (and below)

Reply:
That's actually by intention because if we indent the `<div>` here, the `pre` and `code` element would have some additional whitespace and throw off the docs.

I tried using a HTML minifier for Bazel and tested it, but the whitespaces were preserved. That's why the `pre` and `code` already were indented in that way (we use a minifier at the moment;)